.gitignore

```diff
@@ -20,3 +20,4 @@ deploy/deploy_key.pub
 .vscode/
 docs/docs.json
 webpack/
+.idea/
```

Comment:
Is this relevant to the PR itself? 👀

Reply:
No but I thought since .vscode is already there why not also help people using webstorm as IDE to not have project files alles each time :eyes:

Reply:
added*

Reply:
Because it really should be in a seperate PR.